python/tests/test_lowlevel.py

```diff
@@ -1939,6 +1939,7 @@ def f(mutations):
                     ancestral_state_offset=ancestral_state_offset,
                     metadata=None,
                     metadata_offset=None,
+                    metadata_schema=None,
```

Comment:
Shouldn't need this

Reply:
Yep gone in https://github.com/tskit-dev/tskit/pull/687/commits/682b97da754cb07b00969d8d75c6b06d0314976f